spinetoolbox/spine_db_editor/widgets/stacked_view_mixin.py

```diff
@@ -162,7 +151,18 @@ def _reset_filters(self):
             model.set_filter_class_ids(self._filter_class_ids)
         for model in (self.parameter_value_model, self.entity_alternative_model):
             model.set_filter_entity_ids(self._filter_entity_ids)
-            model.set_filter_alternative_ids(self._filter_alternative_ids)
+            alternatives = self.get_all_alternatives()
+            model.set_filter_alternative_ids(alternatives)
+
+    def get_all_alternatives(self):
+        """Combines alternative ids from Scenario- and Alternative tree selections."""
```

Comment:
I do not think they actually use dash like this in English. I would write such phrases without the dash: "Combines alternative ids from Scenario and Alternative tree selections." Arguably, the Finnish style is nicer, of course ;)